sdk/remoterendering/azure-mixedreality-remoterendering/tests/test_client_async.py

```diff
@@ -210,8 +210,10 @@ async def test_failed_conversion_missing_asset(self, recorded_test, account_info
             await conversion_poller.result()
 
         error_details = excinfo.value
-        assert "invalid input" in error_details.error.message.lower()
-        assert "logs" in error_details.error.message.lower()
+        assert "InputContainerError" == error_details.error.code
```

Comment:
Shouldn't error code a number?

Reply:
Our errors are based on this format: https://learn.microsoft.com/en-us/python/api/azure-core/azure.core.exceptions.odatav4format?view=azure-python

So no, in this case the code isn't a number but an actual string.